draft-irtf-cfrg-vdaf.md

```diff
@@ -3595,6 +3595,118 @@ measurement is sharded. This is provided to the FLP by Prio3.
 | `Field`          | `Field128` ({{fields}})                         |
 {: title="Parameters of validity circuit Histogram."}
 
+### Prio3MultihotCountVec
+
+For this instance of Prio3, each measurement is a vector of ones and zeros,
+where the number of ones is bounded. This provides a functionality similar to
+Prio3Histogram except that more than one entry may be non-zero. This allows
+Prio3MultihotCountVec to be composed with a randomized response mechanism, like
+{{EPK14}}, for providing differential privacy. (For example, each Client would
+set each entry to one with some small probability.)
+
+Prio3MultihotCountVec uses XofTurboShake128 ({{xof-turboshake128}}) as its XOF.
+Its validity circuit, denoted `MultihotCountVec`, uses `Field128` ({{fields}})
+as its default finite field. It has three parameters: `length`, the number of
+of entries in the count vector; `max_weight`, the maximum weight of the vector
+(i.e., the number of non-zero entries); and `chunk_length`, used the same way
+as in {{prio3sumvec}} and {{prio3histogram}}. It is RECOMMENDED to set
+`chunk_length` to an integer near the square root of `length` (see
+{{parallel-sum-chunk-length}}).
+
+The reported weight is encoded in a way that ensures it is in
+`range(max_weight)`. Specifically:
```

Comment:
We shouldn't use `range(max_weight)` Python-style notation here (and elsewhere), as `max_weight` should be an inclusive upper bound, but `range()` has an exclusive upper bound.

Reply:
Hmm, I think that making the VDAF's parameter an exclusive bound on the measurement's weight, rather than an inclusive bound, is confusing to the end user. I'd rather say "I want between zero and five ones, so I'll construct `Prio3MultihotCountVec` with `max_weight = 5`", than "I want between zero and five ones, so I'll construct `Prio3MultihotCountVec` with `weight_bound = 6`". Plus, the reader would have to be familiar with Python to catch the nuance.

Reply:
Hmm, I figured aligning with Python would make things clearer. I don't feel strongly about this though, and since it's what we we already doing, I'm happy to revert.

Reply:
Done - PTAL